r2/r2/models/link.py

```diff
@@ -230,6 +236,7 @@ def _submit(cls, is_self, title, content, author, sr, ip,
             ip=ip,
             comment_tree_version=cls._choose_comment_tree_version(),
             is_self=is_self,
+            over_18=over_18
```

Comment:
Super tiny style nit: we make sure to put a trailing comma on the end of each line, so that if we need to add another new argument it just requires adding the new line, not also adding the comma on the previous line (like how you didn't need to do that here).